File: ueueo-framework/ueueo-eventbus/src/main/java/com/ueueo/eventbus/local/LocalEventMessage.cs
using System;

namespace Volo.Abp.EventBus.Local;

public class LocalEventMessage
{
    public ID MessageId;//  { get; }

    public Object EventData;//  { get; }

    public Type EventType;//  { get; }

    public LocalEventMessage(Guid messageId, Object eventData, Type eventType)
    {
        MessageId = messageId;
        EventData = eventData;
        EventType = eventType;
    }
}

File: ueueo-framework/ueueo-eventbus/src/main/java/com/ueueo/eventbus/local/LocalEventMessage.java
package com.ueueo.eventbus.local;

import com.ueueo.ID;
import lombok.Getter;

@Getter
public class LocalEventMessage {
    private ID messageId;

    private Object eventData;

    private Class<?> eventType;

    public LocalEventMessage(ID messageId, Object eventData, Class<?> eventType) {
        this.messageId = messageId;
        this.eventData = eventData;
        this.eventType = eventType;
    }
}
